cmd/geth/chaincmd.go

```diff
@@ -169,13 +168,7 @@ func initGenesis(ctx *cli.Context) error {
 		if err != nil {
 			utils.Fatalf("Failed to open database: %v", err)
 		}
-		// @NOTE:shyft instantiate BlockExplorerDB here
-		connStr := "user=postgres dbname=shyftdb sslmode=disable"
-		blockExplorerDb, err := sql.Open("postgres", connStr)
-		if err != nil {
-			return nil
-		}
-		_, hash, err := core.SetupGenesisBlock(chaindb, genesis, blockExplorerDb)
+		_, hash, err := core.SetupGenesisBlock(chaindb, genesis)
```

Comment:
Add the NOTE:shyft back please

Reply:
We're not instantiating the blockexplorerdb here so is it really necessary ?

Reply:
I think its good to have it everywhere that we've added any lines of code in exisiting files

Reply:
With this PR there won't be any lines we've added, it'll just be confusing for developers looking for shyftcode

Reply:
ok

Reply:
oh sorry i read that code wrong you removed what we added